Source/JavaScriptCore/runtime/MachineContext.h

```diff
@@ -719,6 +684,64 @@ inline void*& argumentPointer<1>(mcontext_t& machineContext)
 #endif
 }
 
+inline void* wasmInstancePointer(const mcontext_t& machineContext)
```

Comment:
I feel like we have enough of these that it might be worth having a general `gpr` accessor like `Probe::Context`

Reply:
Yes, it can be a good extension in the future.